packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -690,7 +690,11 @@ export class AllocationReceiptCollector implements ReceiptCollector {
         ),
       },
     )
-    return response.data!
+    if (!response.data) {
```

Comment:
This was added in case an undefined or None is returned?
Cause it can return an empty response

Reply:
It should only return empty response in case there was an error. In that case, we want to throw an error.